### DISCUSSION TITLE
Farming on many machines doesn't show total plot count and expect time

### DISCUSSION THREAD

**Discussion by gades** [Chia-Network/chia-blockchain#6562] [Ideas]
According to article https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines is good option to distribute HDD between physical servers but it doesn't show total space and expected time correctly

I see there only local HDD 

Farming status: Farming
Total chia farmed: 0.0
User transaction fees: 0.0
Block rewards: 0.0
Last height farmed: 0
Plot count: 51
Total size of plots: 5.048 TiB
Estimated network space: 19470.702 PiB
Expected time to win: 2 years and 2 months

Also, slave harvester showing wrong info 
Connection error. Check if wallet is running at 9256
Connection error. Check if full node is running at 8555
Connection error. Check if farmer is running at 8559
Farming status: Not available
Total chia farmed: Unknown
User transaction fees: Unknown
Block rewards: Unknown
Last height farmed: Unknown
Plot count: 16
Total size of plots: 1.584 TiB
Estimated network space: Unknown
Expected time to win: Unknown
Note: log into your key using 'chia wallet show' to see rewards for each key

I'm expecting 
Farming status: Farming
Expected time to win:  (time for 16 plots)

on main node it should be 51 + 16 = 67

**Comment by happycouak** [comment]
Duplicates and/or relates with https://github.com/Chia-Network/chia-blockchain/discussions/6467
